internal/service/relationships_test.go

```diff
@@ -54,7 +55,8 @@ func TestRelationshipsService_CreateRelationships(t *testing.T) {
 			expected,
 		},
 	}
-	_, err = relationshipsService.CreateTuples(ctx, req)
+	reqCopy := proto.Clone(req)
```

Comment:
Trying to understand the purpose of clone() here.

Reply:
The app makes changes to the request before processing it, but the tests are reusing the request object for asserts, which ends up breaking. Passing a clone of the request instead keeps them separate.